### ISSUE TITLE
Provide other installation packages

### ISSUE THREAD

**Issue by abnerlee** [typora/typora-issues#1457]
- [ ] [AppImage](https://github.com/typora/typora-issues/issues/583)
       Unofficial Repo -> https://bintray.com/probono/AppImages/Typora

- [ ] [Flatpak](https://github.com/typora/typora-issues/issues/1456) 
       Unofficial Build script -> https://github.com/danieldk/typora-flatpak

- [ ] [RPM](https://github.com/typora/typora-issues/issues/1379)
       Unofficial Build Script -> https://github.com/RPM-Outpost/typora 

- [ ] [Chocolatey](https://github.com/typora/typora-issues/issues/1151)
        Unofficial Repo -> https://chocolatey.org/packages/typora

**Comment by mikeymop** [comment]
Thank you!

**Comment by dixonge** [comment]
I also found this and installed it:

https://snapcraft.io/typora-alanzanattadev

**Comment by skewty** [comment]
SNAP seems preferred over FlatPak because it supports more than just GUI apps (example SystemD services).

(Examples: JetBrains PyCharm, Spotify, PlexMediaServer, NextCloud, VisualStudioCode, VLC, Remmina, FireFox, Brave, ...)

**Comment by idzikovsky** [comment]
+1 for snap

**Comment by wencan** [comment]
+1 for snap

**Comment by mikeymop** [comment]
I would argue for flatpak because snap breaks posix and you will reach less users that way.

Additionally, if you use flatpak you can respect the users theme choices which would make Typora look more native on Linux.

**Comment by codingforfun** [comment]
+1 for AppImage

**Comment by AntoninCamus** [comment]
I agree with @mikeymop on the fact flatpak is more FOSS proof, since visibly snap will stay Ubuntu only, while flatpak is relatively agnostic even if pushed by Gnome / Fedora teams (Flathub is the "default" repo, but elementaryOS which is going to integrate flatpak is going to have his one). 
Furthermore, it has less overhead at start (initial mount of snap is really long), don't depend on Systemd (which makes it more compatible with any OSes), is more interesting on the management of dependencies as well as on the deduplication side.
I don't really see finally the interest of AppImage, since you need as on Windows to ship all of your dependancies (no dependancies mounting), and you need to check your update yourself (no package manager).

I would do a flatpak version as well as a snap version (since it is relatively well integrated into Ubuntu).

**Comment by dastorm** [comment]
Another vote for snap as an official package. The app this way will be available in the ubuntu gui store and will be easily installable on many other distros like fedora and manjaro.

**Comment by mikeymop** [comment]
> easily installable on many other distros like fedora and manjaro.

As a Fedora user, Snap packages do not *just work*.
The reason being is that snap now requires a root level `/snap` directory and a symlink pointing to there for the legacy directory.

From what I have read `/snap` breaks Linux Standard Base and will not be accepted by Fedora. This means users have to modify their system away from what is supported in order to use snaps built with newer versions.


I don't see snap or RedHat budging bit frankly believe it should be Canonical that conforms to the standards.

**Comment by dastorm** [comment]
I'm literally using right now quite a few snaps (mailspring, rambox CE, electronplayer) on fedora 31. It works like in any other distro, the only difference is that the sandboxing isn't as thorough on SElinux as it is with apparmor (but that's being worked on). But please let's keep this issue about typora and not turn it into a snap vs flatpak rant.

**Comment by elQuero93** [comment]
+1 for Flatpak for UI integration and less overhead.

**Comment by syco** [comment]
The link to the flatpak is dead, I made a new one, if anyone needs it: [flatpak](https://github.com/syco/io.typora.flatpak)
It's very easy to push new apps to flathub, you should have a look ;)

**Comment by FilBot3** [comment]
+1 for Flatpak
+1 for RPM (Fedora)

**Comment by meeas** [comment]
@abnerlee just giving you the heads up that @syco 's repo url was changed to the official Flathub organization.  The new URL fo your top post is:

https://github.com/flathub/io.typora.Typora

I'm assuming you are leaving this issue open so people see it easier.  ;-)

**Comment by h0ek** [comment]
+1 for Flatpak
+1 for RPM (Fedora)

**Comment by nozense** [comment]
Anything but deb and snap please.

**Comment by idzikovsky** [comment]
+1 for DEB and Snap

**Comment by h0ek** [comment]
> +1 for DEB and Snap

hi, you can download deb directly or using repo https://typora.io/#linux
